src/twisted/mail/pop3.py

```diff
@@ -855,7 +858,13 @@ def do_PASS(self, password):
 
         @type password: L{bytes}
         @param password: A password.
+
+        @type args: L{tuple} or L{None}
+        @param args: Other words composing the password.
```

Comment:
It can't actually be `None` because it's `*args`.  It can be any tuple, including an empty tuple.